core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -628,7 +628,25 @@
     - cluster: default
       namespace: ci
       name: cluster-secrets-metal
-# cluster-secrets-azure is not synced. Need to ask DPP team for this.
+#azure4
+- from:
+    pull-secret:
+      bw_item: os4-installer.openshift-ci.azure
```

Comment:
@stevekuznetsov 

This one in production is not the same as `pull-secret` in  `quay.io`. I do not know where it came from.
So I add an attachment `pull-secret` into BW-item `os4-installer.openshift-ci.azure`.

Reply:
Deleted the added attachment ^^^ and use the one from quay.io.

The pull-secret in quay.io has one more entry for `quay.io/openshift-cnv-dev`.

I guess the reason of the unsync came from the fact that `azure4` has never been under our BW-shell since it is created by `make`. After its creation, we updated the BW item with a new auth entry.

So in my PC, I run the tool with `--force=true` and now it uses the same pull secret as others.